app/src/ingest_edd_web.py

```diff
@@ -32,18 +32,47 @@ def _ingest_edd_web(
         urls_processed.add(item["url"])
 
         content = item.get("main_content", item.get("main_primary"))
-        assert content, f"Item {name} has no main_content or main_primary"
+        if not content:
+            logger.warning("Item %s has no main_content or main_primary", name)
+            continue
```

Comment:
This should not occur. If it does, LMK which URL.

Reply:
Here's one:
> Processing EDD Online Application Now Available in Top Eight Languages  (https://edd.ca.gov/en/about_edd/news_releases_and_announcements/news-24-12/)

Reply:
I have content for that:
```
"main_content": "[Español](/es/about_edd/news_releases_and_announcements/edd-online-application-now-available-in-top-eight-languages/)\n\n[Print](javascript:print();)\n\n# EDD Online Application Now Available in Top Eight Languages\n\nPublished: March 28, 2024\n\nNR No. 24-12  \nContact: Loree Levy/Aubrey Henry  \n916-654-9029  \n[mediainquiries@edd.ca.gov](mailto:mediainquiries@edd.ca.gov)\n\n**What You Need to Know:** California’s Armenian, Korean, and Tagalog communities can now apply online for unemployment benefits in their preferred language. This enhancement adds to English, Spanish, [Simplified Chinese, Traditional Chinese, and Vietnamese](https://edd.ca.gov/en/about_edd/news_releases_and_announcements/news-23-48/) languages already available. Together, these eight languages represent the top spoken languages of working-age adults in California.\n\n**SACRAMENTO**– Today, the Employment Development Department (EDD) announced ...
```
Retry the scraping from `main` -- you probably run the older code.

Reply:
You're right, rescraping fixed it. Will restore the assert!